modules/data-entry/src/main/frontend/src/dataHomepage/ExportButton.jsx

```diff
@@ -0,0 +1,395 @@
+                >
+                  <FormControlLabel value=".csv" control={<Radio />} label=".csv" />
+                  <FormControlLabel value=".tsv" control={<Radio />} label=".tsv" />
+                </RadioGroup>
+              </Grid>
+            </Grid>
+
+            <Grid container alignItems='center' direction="row" className={classes.container}>
+              <Grid item xs={4}><Typography variant="subtitle2">Header format:</Typography></Grid>
+              <Grid item xs={8}>
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={labelFormat}
+                      onChange={(event) => { setLabelFormat(event.target.checked);}}
+                      name="labels"
+                    />
+                  }
+                  label="Labels"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={identifiersFormat}
+                      onChange={(event) => { setIdentifiersFormat(event.target.checked);}}
+                      name="identifiers"
+                    />
+                  }
+                  label="Identifiers"
+                />
+              </Grid>
+            </Grid>
+
+            <Grid container alignItems='center' direction="row" className={classes.container}>
+              <Grid item xs={4}><Typography variant="subtitle2">Data format:</Typography></Grid>
+              <Grid item xs={8}>
+                <RadioGroup
+                  row
+                  defaultValue="values"
+                  aria-label="data"
+                  name="data"
+                  value={hasLabeles}
+                  onChange={(event) => setHasLabeles(event.target.value === "true")}
+                >
+                  <FormControlLabel value={true} control={<Radio />} label="Labels" />
+                  <FormControlLabel value={false} control={<Radio />} label="Values" />
+                </RadioGroup>
+              </Grid>
+            </Grid>
+
+            <Grid container alignItems='center' direction="row" className={classes.container}>
+              <Grid item xs={4}><Typography variant="subtitle2">Column selection mode:</Typography></Grid>
+              <Grid item xs={8}>
+                <RadioGroup
+                  row
+                  defaultValue="isinclude"
+                  aria-label="isinclude"
+                  name="isinclude"
+                  value={isInclude}
+                  onChange={(event) => setIsInclude(event.target.value === "true")}
+                >
+                  <FormControlLabel value={true} control={<Radio />} label="Include" />
+                  <FormControlLabel value={false} control={<Radio />} label="Exclude" />
+                </RadioGroup>
+              </Grid>
+            </Grid>
+
+            <Grid container alignItems='center' direction="row" className={classes.container}>
```

Comment:
`alignItems='center'` looks great on all other entries except this one which can get very tall. It should be `start` and it should be combined with some styling for better top alignment.
```suggestion
            <Grid container alignItems='start' direction="row" className={classes.container}>
```